### DISCUSSION TITLE
file->json处理->kafka，message 过长处理成了【...】结尾

### DISCUSSION THREAD

**Discussion by Jalyn-X** [alibaba/ilogtail#1211] [Help] [answered]
![image](https://github.com/alibaba/ilogtail/assets/142231844/bafabd66-b031-4dbf-8eec-4d4bf956d126)
![image](https://github.com/alibaba/ilogtail/assets/142231844/1ddd2bc1-d196-423b-9f6c-ecff9806b16e)

看了 file_log、json、kafkav2 的文档，没有留意到是哪个参数的限制。
麻烦可以说明下是哪个参数做了控制吗？感谢！

**Comment by Jalyn-X** [comment]
有知道如何处理的嘛

**Reply by linrunqi08** [reply]
现在的长度是多少呢

**Reply by Jalyn-X** [reply]
经测试，1w字节是可以的，10w字节就显示省略号了，没有去测试临界值。

**Reply by linrunqi08** [reply, marked as answer]
@Jalyn-X  你这个...是从哪里看到的呢？logtail内部应该没有10000的限制参数。
要排查的话
1. 可以先把kakfa插件换成flusher_stdout，写入到文件中看下，确认下flusher之前的处理没问题
2. kakfa插件里也可以加日志看下

**Reply by Jalyn-X** [reply]
@linrunqi08 
我的，是日志框架层面的限制。
